src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -176,6 +176,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
```

Comment:
I don't believe this is typically used on hot paths, and after some brief searching in ASP.NET, I don't see much in the way of usage there.  Until it's proven as a bottleneck, I think it's fine to hold off in order to avoid this dependency for all of System.Runtime.Extensions (not clear why WebUtility is here, actually).